### ISSUE TITLE
Gone after locking screen

### ISSUE THREAD

**Issue by leflamm** [ubuntu/gnome-shell-extension-appindicator#111]
Hi,

whenever I lock my screen and log in again the KStatusNotifierItem extension is gone. I can bring it back to life by doing a Gnome Shell quick reload: [Alt]+[F2] "r"

I'm using Ubuntu 17.10

**Comment by jhasse** [comment]
Can you have a look if `journalctl -e /usr/bin/gnome-shell` shows any error messages?

**Comment by leflamm** [comment]
Full of those messages:
```
Nov 16 10:16:39 <hostname> gnome-shell[1846]: _st_paint_shadow_with_opacity: assertion 'shadow_pipeline != NULL' failed
```

**Comment by jhasse** [comment]
Can you try with all extensions disabled except for KStatusNotifierItem?

**Comment by leflamm** [comment]
Sorry for the delay. Disabled all other extensions ... still same effect.

**Comment by marlid91** [comment]
I've the same problem.  It also disappears if I disable other extensions. Do you guys have any updates on this?

**Comment by ssweeny** [comment]
I was able to fix this on my Ubuntu 18.04 system by killing `indicator-application-service` and removing the `indicator-application` package. Seems that service (a holdover from Unity) prevents the appindicator extension from loading properly.

**Comment by marlid91** [comment]
Thanks! That seems to have fixed it. Where could we report this bug to Canonical?
